core/src/main/java/com/datastax/oss/driver/internal/core/metrics/DropwizardMetricUpdater.java

```diff
@@ -33,46 +40,80 @@
   private static final Logger LOG = LoggerFactory.getLogger(DropwizardMetricUpdater.class);
 
   protected final Set<MetricT> enabledMetrics;
+
   protected final MetricRegistry registry;
 
-  protected DropwizardMetricUpdater(Set<MetricT> enabledMetrics, MetricRegistry registry) {
+  protected final Cache<String, Metric> metricsCache;
+
+  protected DropwizardMetricUpdater(
+      Set<MetricT> enabledMetrics, MetricRegistry registry, Ticker ticker) {
+    this(enabledMetrics, registry, ticker, Duration.ofHours(1));
```

Comment:
`Duration.ofHours(1)` - this will be parameterized (to be able to set via `reference.conf`)